src/views/hostnetworkconfiguration/components/TopologySidebar/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { FC } from 'react';
+
+import {
+  Checkbox,
+  Flex,
+  FlexItem,
+  List,
+  ListItem,
+  Stack,
+  StackItem,
+  Title,
+} from '@patternfly/react-core';
+import { NodeNetworkConfigurationInterface } from '@types';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+import { getPorts } from '@utils/interfaces/getters';
+import { getSystemName } from '@utils/neighbors/getters';
+
+import NeighborInformations from './NeighborInformations';
+
+type InterfaceDrawerDetailsTabProps = {
+  selectedInterface: NodeNetworkConfigurationInterface;
+};
+
+const InterfaceDrawerDetailsTab: FC<InterfaceDrawerDetailsTabProps> = ({ selectedInterface }) => {
+  const { t } = useNMStateTranslation();
+  const ports = getPorts(selectedInterface);
+
+  return (
+    <Stack hasGutter>
+      {selectedInterface.vlan && (
```

Comment:
Is `selectedInterface` guaranteed to exist and have all of these fields? If not, we should probably add optional chaining operators on all of these calls.